style.css

```diff
@@ -0,0 +1,73 @@
+section {
+    margin:auto;    
+    width: 625px;
+    background-color: lightgray;
+    color: white;
+}
+
+article {
+    display: inline-block;
```

Comment:
This doesn't really do anything, you can set widths of block level elements as well